packages/jaspr/lib/src/browser/component_anchors.dart

```diff
@@ -0,0 +1,99 @@
+import 'dart:async';
+import 'dart:convert';
+
+import 'package:universal_web/web.dart' as web;
+
+import '../../jaspr.dart';
+import 'options.dart';
+
+sealed class ComponentAnchor {
+  ComponentAnchor(this.name, this.startNode);
+
+  final String name;
+  final web.Node startNode;
+  late web.Node endNode;
```

Comment:
Since this can be a bit risky, consider adding a doc comment describing when this should be assigned (and therefore can be accessed).

Reply:
This is guaranteed to be assigned before being used.